Sources/SwiftRedis/Redis.swift

```diff
@@ -23,7 +23,7 @@ import Foundation
 public class Redis {
     
     /// Redis Serialization Protocol handle
-    private var respHandle: RedisResp?
+    public var respHandle: RedisResp?
```

Comment:
Why do you need to change the visibility?

Reply:
To access socket.